docs/MO_DG/prepare_model/convert_model/onnx_specific/Convert_RNNT.md

```diff
@@ -0,0 +1,102 @@
+# Convert ONNX* RNNT to the Intermediate Representation {#openvino_docs_MO_DG_prepare_model_convert_model_onnx_specific_Convert_RNNT}
+
+This instruction covers conversion of RNN-t model from [MLCommons](https://github.com/mlcommons) repository. Before
+converting into IR PyTorch model need to be exported into ONNX. In order to that follow the steps below.
+
+**Step 1**. Clone RNN-t PyTorch implementation from MLCommons repository. In order to pull only RNN-t model 
+without full repository make a shallow clone. If you alreade have a full repository skip this and go to **Step 2**:
```

Comment:
In order to pull only RNN-t model without full repository make a shallow clone ---> Make a shallow clone to pull only RNN-T model without full repository.